lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
+    return false;
+  }
+  if (!args.every(n => Node.ConstantTerms.isConstantTerm(n))) {
```

Comment:
Okay, I've been looking into this too and I'm not quite sure how to rewrite it. I'm not sure than an helper function will be enough, because I have to contruct the power-node somehow and this is currently done in `ConstantTerm` since it cant be done in `PolynomialTerm`. But i agree about coefficient thing, that's not needed.